core/templates/pages/story-viewer-page/story-viewer-page.component.ts

```diff
@@ -178,6 +210,38 @@ export class StoryViewerPageComponent implements OnInit {
     // path.
     this.pathIconParameters = [];
   }
+
+  isHackyStoryTitleTranslationDisplayed(): boolean {
+    return (
+      this.i18nLanguageCodeService.isHackyTranslationAvailable(
+        this.storyTitleTranslationKey) &&
```

Comment:
Same notes as above for indentation, here and below.

Reply:
Done!